Cubical/HITs/Ints/QuoInt/Base.agda

```diff
@@ -8,17 +8,60 @@ open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.Equiv
 open import Cubical.Foundations.Transport
 open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+open import Cubical.Relation.Nullary
 
-open import Cubical.Data.Int hiding (neg; abs; sgn)
-open import Cubical.Data.Nat hiding (elim)
+open import Cubical.Data.Int as Int using (Int; sucInt; predInt; discreteInt; isSetInt)
+open import Cubical.Data.Nat as ℕ using (ℕ; zero; suc)
+open import Cubical.Data.Bool as Bool using (Bool; not; notnot)
 
 variable
   l : Level
 
+
+Sign : Type₀
```

Comment:
Was there any benefit to using Bool for Sign instead of the old Sign datatype?

Reply:
Yes, I didn't have to redefine `⊕`, `⊕-comm`, and `notnot` for `Sign`. I figured using `Bool` here was a nice way to get rid of some obvious code duplication.